forge/routes/api/project.js

```diff
@@ -360,7 +360,7 @@ module.exports = async function (app) {
             )
             reply.send({ status: 'okay' })
         } catch (err) {
-            reply.code(500).send({ error: err.toString() })
+            reply.code(400).send({ code: 'unexpected_error', error: err.toString() })
```

Comment:
```suggestion
            reply.code(500).send({ code: 'unexpected_error', error: err.toString() })
```
Sorry, should have tagged all of the ones that really are backend errors